recipe/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set name = "antlr" %}
-{% set version = "4.8" %}
-{% set sha256 = "" %}
+{% set major_version = "4" %}
+{% set minor_version = "8" %}
+{% set version = major_version + "." + minor_version %}
+{% set revision = "1" %}
```

Comment:
anyhow is perhaps a bridge too far... the bot _really_ likes there to be a jinja var called `version` which is a string, so that it can try updating it when the upstream changes. Also, there have historically been `x.y.z` releases, so this would need to be fixed again, again...

Reply:
Yeah, agreed. This is mostly a remnant of me trying to figure things out before understanding the maven issue fully. Will clean up.